packages/api-gateway/src/drivers/http/routes/admin_panel/api/index.js

```diff
@@ -1,8 +1,8 @@
 const { adminPanelAdapters } = require('../../../adapters');
-// const schema = require('./schema');
+const schema = require('./schema');
 
 async function adminPanelRouter(fastify) {
   await fastify.get('/', adminPanelAdapters.sayHello);
+  await fastify.patch('/user_status/:user_id', { schema }, adminPanelAdapters.changeUserStatus);
```

Comment:
Can you replace "_" with "-"? URLs normally use "kebab case", that means that it should be lowercase and "-" instead of spaces.

```suggestion
  await fastify.patch('/status/:user_id', { schema }, adminPanelAdapters.changeUserStatus);
```